modules/util/net.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package util
+
+import (
+	"net"
+)
+
+// IsIPPrivate for net.IP.IsPrivate. TODO: replace with `ip.IsPrivate()` if min go version is bumped to 1.17
```

Comment:
1.16.x branch won't be bumped to go version 1.17

```suggestion
// IsIPPrivate for net.IP.IsPrivate.
```